backend/app/main.py

```diff
@@ -174,6 +175,33 @@ async def shatter_parent(
     return result
 
 
+@app.patch(
```

Comment:
H: Sorry forgot to mention this before but could you add a test for this function, too?

Reply:
Ah, yes 100%. Had this on my list todo but totally forgot!